.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: Solid Data Wallet CI
+
+on:
+  pull_request: {}
+  push:
+    branches: [ main ]
+
+env:
+  CI: true
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    uses: inrupt/typescript-sdk-tools/.github/workflows/reusable-lint.yml@v3
+
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+          cache: npm
+      - run: npm ci
+      - run: npm run test:snapshot
+      # Upload coverage for sonarqube
+      - uses: actions/upload-artifact@v4
+        with:
+          name: code-coverage
+          path: coverage/
+
+  sonarqube:
```

Comment:
Might just call this `sonar`

Reply:
I will now that this is internally focussed. The default proposed by Sonar is sonarqube